src/index.html

```diff
@@ -5,8 +5,260 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header header">
+      <div class="header__logo">
+        <a href="#">
+          <img src="images/logo.png" alt="moyo-icon">
+        </a>
+      </div>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link-active is-active">Apple</a>
```

Comment:
active це модифікатор, тому його потрібно записати так
```suggestion
            <a href="#" class="nav__link nav__link--active is-active">Apple</a>
```